sdk/beatoven_sdk/client.py

```diff
@@ -0,0 +1,80 @@
+from .api import get_track, list_tracks, create_track, compose_track, get_task_status
+import asyncio
+
+
+def extract_track_info(track):
+    track_info = {}
+    track_info["title"] = track["title"]
+    track_info["id"] = track["project_id"]
+    return track_info
+
+
+async def list_all_tracks():
+    tracks_data = await list_tracks()
+    return list(map(extract_track_info, tracks_data["tracks"]))
+
+
+async def get_track_by_id(track_id):
```

Comment:
Also i think this might always return None for track_url since it wasnt downloaded

Reply:
The `track_url` is the URL to download the track from, so that would be the storage bucket path. I think it could return `None` if the track is still being created, but once it is created a URL should always exist.

Reply:
Added an integration test to verify [e1a0749](https://github.com/Beatoven/public-api/pull/5/commits/e1a074950ea0fcbfb497f27054cad321b0d209d1)